src/Angor/Client/Pages/Create.razor

```diff
@@ -847,6 +849,21 @@
 
         try
         {
+            // Check if the FounderKey is in use and get the total keys used
+            var keyCheckResult = await FounderKeyService.CheckFounderKeyAsync(project.ProjectInfo.FounderKey);
```

Comment:
Putting this in the `CreatProjectTransaction` is probably not the right thing to do, we must know much before this point if the key was already used

Reply:
We do rescan before creating,
But there's a scenario this scenario.
1. Logged on browser A, B.
2. start create browser A.
3. Start create browser B.
4. Create browser B.
5. Create Browser A.

unless you put a rescan when creating, you won't know that there's another project created on the same key.
but also now that i think of it we will need to refresh the data in the create a project. (almost everything except project info no?)

I know that this is an edge case that is not expected (especially when we suspect that per user there will be probably only 1 project), but still this should be addressed in some way because we cannot have any loophole of over riding a project.

Reply:
We need to have a meeting about trusting the relays for knowing if the project was created or only look at the indexer to see if a transaciton exists

Reply:
Okay, then i'll wait with this PR until we do that,
We should align on how much we trust the relays, and on which instances we can trust it.

Also, according to that we should check if there are other places in angor that trust the relay instead of the indexer?